src/functions/lookupRandomCocktail.js

```diff
@@ -0,0 +1,25 @@
+const dotenv = require("dotenv");
```

Comment:
This file name looks good. very self-descriptive.

Reply:
Thank you!